docs/computing/containers/creating.md

```diff
@@ -1,26 +1,26 @@
 
-If you cannot find a pre-built container, one option is to build your own. If a Docker container image already exists, you can often simply convert that to a Singularity container. Another option is to build your own container from scratch.  Both approaches will be discussed below. As always, if you have any problems or questions, don't hesitate to contact [CSC's Service Desk](https://www.csc.fi/en/contact-info).
+If you cannot find a pre-built container, one option is to build your own. If a Docker container image already exists, you can often simply convert that to an Apptainer container. Another option is to build your own container from scratch.  Both approaches will be discussed below. As always, if you have any problems or questions, don't hesitate to contact [CSC's Service Desk](https://www.csc.fi/en/contact-info).
 
 ## Converting a Docker container
 
-If you already have an existing Docker container, in many cases it can easily be converted to a Singularity image. Docker container images can be found in public repositories such as [Docker Hub](https://hub.docker.com/), but **please take care to only use images uploaded from reputable sources** as these images can easily be a source of security vulnerabilities or even contain malicious code.
+If you already have an existing Docker container, in many cases it can easily be converted to an Apptainer image. Docker container images can be found in public repositories such as [Docker Hub](https://hub.docker.com/), but **please take care to only use images uploaded from reputable sources** as these images can easily be a source of security vulnerabilities or even contain malicious code.
 
 GPU-optimized containers can also be found in [NVIDIA's GPU cloud (NGC)](https://ngc.nvidia.com/). These containers have been prepared by NVIDIA, and should thus be safe.
 
-Further [information about converting Docker containers to Singularity can be found in the Singularity documentation](https://sylabs.io/guides/3.6/user-guide/singularity_and_docker.html).
+Further [information about converting Docker containers can be found in the Apptainer documentation](https://apptainer.org/docs/user/main/docker_and_oci.html).
 
-Here is an example of how to build a Singularity image from [NVIDIA's PyTorch Docker image](https://ngc.nvidia.com/catalog/containers/nvidia:pytorch). We'll use `sinteractive` as heavy processing should not be done in the login nodes.
+Here is an example of how to build an Apptainer image from [NVIDIA's PyTorch Docker image](https://ngc.nvidia.com/catalog/containers/nvidia:pytorch). We'll use `sinteractive` as heavy processing should not be done in the login nodes.
 
 ```bash
-# Let's start a 1 hour interactive job
-sinteractive --account <project> --time 1:00:00
+# Let's start a 1 hour interactive job with 16G of memory
+sinteractive --account <project> --time 1:00:00 -m 16G
```

Comment:
With my testing `--tmp 64` is needed on Puhti. This instructions don't work on Mahti. This is just an example so I don't think there is any need to change it, maybe just add "We'll use `sinteractive` on Puhti..."  to specify that these were run on Puhti.

Reply:
I added `--tmp 64` and a mention that the example is for Puhti.